Model/Insight/Params.php

```diff
@@ -149,4 +152,36 @@ public function getParamsArray()
 
         return $params;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getBatchQuery()
+    {
+        $params="?";
```

Comment:
You can use http://php.net/manual/en/function.http-build-query.php to generate a proper query string